design_temp/clean.py

```diff
@@ -0,0 +1,147 @@
+import csv
+import re
+from dataclasses import dataclass
+
+
+# This script generates design_temp_by_county.csv
+# from the raw text copied from the ENERGY STAR pdf located at:
+# https://www.energystar.gov/ia/partners/bldrs_lenders_raters/downloads/County%20Level%20Design%20Temperature%20Reference%20Guide%20-%202015-06-24.pdf
+
+
+def remove_junk(content):
+    table_title = (
+        "Exhibit 1: Cooling and Heating Design Temperatures by State and County"
+    )
+    content = content.split(table_title)[1]
+    pattern = r"[ ]+ENERGY STAR Certified Homes\nCounty-Level Design Temperature Reference Guide"
```

Comment:
I'm assuming the space characters come from the text representation of the Energy Star graphic. Maybe we shouldn't assume it will always be spaces, but is `.*`?

Reply:
I'm a little nervous that `.*` will accidentally match some useful data that we need to process